app/javascript/src/components/Dashboard/Contacts/Filter/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+
+import { Plus, Search, Settings } from "@bigbinary/neeto-icons";
+import { Typography } from "@bigbinary/neetoui/v2";
+import { MenuBar } from "@bigbinary/neetoui/v2/layouts";
+
+const Filter = ({ isMenuOpen }) => {
```

Comment:
We can just keep the file as `Filter.jsx`. No need to do `Filter/index.jsx` unless you're doing file splitting for the component.